packages/itwinui-react/src/core/Overlay/Overlay.tsx

```diff
@@ -117,7 +117,7 @@ const useInertPolyfill = () => {
         !loaded.current &&
         !isUnitTest
       ) {
-        await new Function(`return import("${modulePath}")`)();
+        await new Function('url', 'return import(url)')(modulePath);
```

Comment:
it shouldn't make a difference here because this does not accept user input, but avoiding interpolation ensures that `modulePath` cannot be used to inject nefarious code

see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/Function#description